odk1-src/launch-apps-from-collect.rst

```diff
@@ -81,3 +82,16 @@ XLSForm
 The ``intent`` attribute is only used when the group has an ``appearance`` of ``field-list``. The format and the functionality of the ``intent`` value is the same as above. If bundle of values returned by the external application contains values whose keys match the type and the name of the sub-fields, then these values overwrite the current values of those sub-fields.
 
 The external app is launched with the parameters that are defined in the intent string plus the values of all the sub-fields that are either text, decimal, or integer. Any other sub-field is invisible to the external app.
+
+.. _launch-apps-uri-data:
+
+Specifying a URI as intent data
+---------------------------------
+
+Since Collect v1.16.0, it is possible to specify a URI that will be used as the data for the intent using the reserved ``uri_data`` parameter. This URI should include a scheme so that external applications will be able to identify the type of content to act on. That is, ``mailto:example@example.com`` is a valid URI but ``example@example.com`` is not.
```

Comment:
> it is possible to specify a URI that will be used as the data for the intent using the reserved ``uri_data`` parameter. 

This sentence confused me. I think it is the `used...using` and (i don't know what to call it) daisy chain handoff of subject/predicate.

(Also, the style guide prefers "You can..." to "It is possible..." cf: https://docs.opendatakit.org/docs-style-guide/#direct-address)

Moreover. I don't know, it was just kind of unclear. I think it is the front-loading of information (URI requiring a scheme before any examples).

Maybe something like...

```
you can specify a URI in the extra parameters using the reserved name ``uri_data``. 
This is particularly useful when using an implicit intent such as `sendto `_, which launches different applications for different types of URIs. 

.. _sendto: https://developer.android.com/reference/android/content/Intent#ACTION_SENDTO

For example: 

``ex:android.intent.action.SENDTO(uri_data='smsto:5555555', sms_body=${message})``
  Launches a new message in an SMS app with the destination number set to ``5555555`` and the message body set to the contents of the ``message`` field.

``ex:android.intent.action.SENDTO(uri_data='mailto:example@example.com?subject=${subject}&body=${message})``
  Launches a new message in an email app with destination address set to ``example@example.com``, the subject set to the contents of the ``subject`` field and the body set to the contents of the ``message`` field.

``ex:android.intent.action.DIAL(uri_data='tel:5555555')``
  Launches a phone dialer with the number ``5555555`` as the number to dial.

Notice that the URI must include a `scheme`_, such as ``mailto:`` or ``https://``.

.. _scheme: https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

```

Notice that I changed your list to a definition list. In general if a list has an item:value format, definition lists are more clear.

If you want to keep the bullets, I prefer an em-dash (wrapped in spaces) instead of a colon because I think it is more clear (since it provides more horizontal separation).

(I don't think we've solidified that preferences in the style guide, though. So if you feel strongly about colons, that's probably fine.)